docs/src/routes/examples/[...2]uniswap-widget/+page.md

```diff
@@ -66,7 +66,11 @@ const degenChain = {
   rpcUrl: 'https://rpc.degen.tips'
 }
 
+<<<<<<< HEAD
```

Comment:
merge conflicts still need to be resolved

Reply:
ahh good catch!